src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/NativeMarshallingAttributeAnalyzerTests.cs

```diff
@@ -0,0 +1,260 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+using static Microsoft.Interop.Analyzers.NativeMarshallingAttributeAnalyzer;
+
+using VerifyCS = LibraryImportGenerator.UnitTests.Verifiers.CSharpAnalyzerVerifier<
+    Microsoft.Interop.Analyzers.NativeMarshallingAttributeAnalyzer>;
+
+namespace LibraryImportGenerator.UnitTests
+{
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/60650", TestRuntimes.Mono)]
+    public class NativeMarshallingAttributeAnalyzerTests
+    {
+        [Fact]
+        public async Task NativeMarshallingWithNullMarshallerType_ReportsDiagnostic()
```

Comment:
Thoughts on removing `NativeMarshalling`/`CustomMarshaller` from all these test method names, since they are already in test classes specifically for `NativeMarshallingAttribute`/`CustomMarshallerAttribute`?